src/components/EthPriceCard.tsx

```diff
@@ -1,90 +1,11 @@
 import React, { useState, useEffect } from "react"
-import styled from "@emotion/styled"
 import axios from "axios"
 
 import Translation from "./Translation"
-import Icon from "./Icon"
 import Link from "./Link"
 import Tooltip from "./Tooltip"
-
-const InfoIcon = styled(Icon)`
-  margin-left: 0.5rem;
-  fill: ${(props) => props.theme.colors.text200};
-`
-
-const Card = styled.div<{
-  isLeftAlign: boolean
-  isNegativeChange: boolean
-}>`
-  display: flex;
-  flex-direction: column;
-  align-items: ${(props) => (props.isLeftAlign ? `flex-start` : `center`)};
-  justify-content: space-between;
-  width: 100%;
-  max-width: 420px;
-  max-height: 192px;
-  background: ${(props) =>
-    props.isNegativeChange
-      ? props.theme.colors.priceCardBackgroundNegative
-      : props.theme.colors.priceCardBackgroundPositive};
-  border-radius: 4px;
-  border: 1px solid
-    ${(props) =>
-      props.isNegativeChange
-        ? props.theme.colors.priceCardBorderNegative
-        : props.theme.colors.priceCardBorder};
-  padding: 1.5rem;
-`
-
-const Title = styled.h4`
-  margin: 0;
-  font-size: 0.875rem;
-  line-height: 140%;
-  letter-spacing: 0.04em;
-  text-transform: uppercase;
-  color: ${(props) => props.theme.colors.text200};
-`
-
-const Price = styled.div<{
-  hasError: boolean
-}>`
-  line-height: 1.4;
-  font-weight: 400;
-  margin: ${(props) => (props.hasError ? `1rem 0` : 0)};
-  font-size: ${(props) => (props.hasError ? props.theme.fontSizes.m : `3rem`)};
-  color: ${(props) =>
-    props.hasError ? props.theme.colors.fail : props.theme.colors.text};
-`
-
-const ChangeContainer = styled.div<{
-  isLeftAlign: boolean
-}>`
-  width: 100%;
-  display: flex;
-  align-items: center;
-  justify-content: ${(props) => (props.isLeftAlign ? `flex-start` : `center`)};
-  min-height: 33px; /* prevents jump when price loads*/
-`
-
-const Change = styled.div<{
-  isNegativeChange: boolean
-}>`
-  font-size: 1.5rem;
-  line-height: 140%;
-  margin-right: 1rem;
-  color: ${(props) =>
-    props.isNegativeChange
-      ? props.theme.colors.fail300
-      : props.theme.colors.success};
-`
-
-const ChangeTime = styled.div`
-  font-size: 0.875rem;
-  line-height: 140%;
-  letter-spacing: 0.04em;
-  text-transform: uppercase;
-  color: ${(props) => props.theme.colors.text300};
-`
+import { Box, Flex, Heading, Icon } from "@chakra-ui/react"
+import { MdInfoOutline } from "react-icons/md"
```

Comment:
import order

move these imports to the top of the file, next to the `react` import

Reply:
@pettinarip shouldn't our linter reformat this automatically?